src/java/org/apache/cassandra/cql3/statements/schema/CreateKeyspaceStatement.java

```diff
@@ -133,6 +133,11 @@ public String toString()
             clientWarnings.add(msg);
         }
 
+        if (attrs.hasProperty("graph_engine"))
```

Comment:
Should we just loop through the obsoleteKeywords here and print a generic WARN for them?  Rather than having keyword specific messages?